### DISCUSSION TITLE
Honeycomb Storage Wall (HSW) adapter

### DISCUSSION THREAD

**Discussion by boc-tothefuture** [aderusha/DDD-Printable-Wall-Control-System#85] [Ideas]
Has anyone tried to make an adapter to support the [Honeycomb Storage Wall](https://www.printables.com/model/152592-honeycomb-storage-wall) accessories?  

The Wall Control system in general is a better fit for the space I am in (stud cavities without drywall).  However, there are a lot of models available for things I would put on the wall that already exist for the honeycomb storage wall. 

I think what would be needed with would be flat left/right pieces that were 8mm depth to match the HSW dimension, then centerpieces of various dimensions that matched the honeycomb pattern.  

Before I play around, I was wondering if anyone had already tried this?

**Comment by aderusha** [comment]
Seems easy enough, something like this? https://github.com/aderusha/DDD-Printable-Wall-Control-System/tree/main/Centerpieces/Honeycomb_Storage_Wall

**Comment by boc-tothefuture** [comment]
Awesome, thanks!  I will print it and let you know if it works.

**Comment by aderusha** [comment]
Added a 3rd option and dropped it on Printables here: https://www.printables.com/model/802796-ddd-wall-control-honeycomb-storage-wall

**Comment by boc-tothefuture** [comment]
I printed and tested.. Worked great!

**Comment by boc-tothefuture** [comment]
Thanks again! 

A couple of observations as I try and use these a bit more. 

I seem to have a lot of spots where I want a single honeycomb.  Printing any of the sizes that we have results in a fair amount of waste because often much of the honeycomb is covered by whatever I am attaching to it. 

It would be helpful (at least in my use case), to have a 1U (or maybe 2U vertical - but only if it fits) honeycomb.   And to have it in two types. 

Type 1 is what you have with the 20mm hexagons and the different thickness to create a place fro the clip to lock into.

Type 2 (and this only makes sense for a 1U or 2U unit) is to have instead the hexagons be the 15.47 by 13.4 that you see in the [Fasteners and parts for attaching hooks and objects](https://www.printables.com/model/152592-honeycomb-storage-wall) section of the spec. The reason being that many (but not all) attachments go into that smaller shape (called an insert).  You don't want to make them all that same size for larger honeycombs because it would consume too much filament. But if you are printing a 1U, then there are pretty good odds you will need that size, so no need to print that and then an insert to snap in.. 




This is a minor optimization, but I think you need the firm structure on the left and ride side for attachment purposes, but I think the tops and bottoms of the existing models could be half hexagons to reduce filament and print time.

**Comment by boc-tothefuture** [comment]
One more question, now that I am thinking about it.. (And I don't know if the math lines up, I will have to look).  Instead of generating the various honeycomb sizes in 6x6, 4x4, etc with the pins.  Would it make sense to just generate some vertical strips that are only 1 honeycomb wide with the pins that connect into the flat bars?  That way you could use all the existing honeycomb generators  and then use these bars + [standard honeycomb joiners](https://www.printables.com/model/152592-honeycomb-storage-wall/files#preview:file-KGGCU) to attach them? 

That way this project doesn't need to worry about generating various honeycomb sizes?  

I think an even more interesting option might be take something like the [standard joiner](https://www.printables.com/model/152592-honeycomb-storage-wall/files#preview:file-KGGCU) and have one side be what it is now, but engineer the other side to be a wall control hook.

**Comment by aderusha** [comment]
The core challenge here is that the wall control system uses 1" spacing, so you have to somehow align that with the honeycomb which ... isn't.  I don't use the honeycomb system at all so I'm just completely guessing based on your descriptions above and at this point, I'm kinda lost.

Do you have any way to maybe sketch this out?  Literal napkin would do, I'm not real clear on the ideas being presented here, but I'm into helping out if I can get a starting point!

**Reply by boc-tothefuture** [reply]
Yes.. I agree.. it is a bit of a stream of consciousness.  Let me toss a crude diagram together.

**Comment by boc-tothefuture** [comment]
Quick follow-up.. you are right, I played around with the standard joiner idea a bit and there are no common/repeated spots that would be the same distance from the slot to the next hexagon.  

I will draw out some of the others, I think it will be helpful to have many sizes that are 1U height (wall control units - not honeycomb units), in various widths 1U, 3U, 4U, 5U, etc (wall control units).

**Reply by aderusha** [reply]
That's the key problem. I calculated it out, the hexagons are staggered so you need to match up a span of two hexagons across which is 42.58mm vs the 25.4mm spacing of Wall Control.  The smallest single panel that would effectively tile the two systems is 54076.6mm across.

I'm going to need a bigger printer for that one :)

**Reply by boc-tothefuture** [reply]
:-)   That would be a nice size printer indeed!  

Yeah, what I am figuring out (slowly, it seems).. is that it really isn't about tiling the two systems.  And doing so doesn't make sense.  A benefit of HSW is that you can move things around as your system changes just like you can with wall control.

And because of that, it doesn't really make sense to move things around a honeycomb on a wall control panel.  It is a more efficient use of both filament and wall control space to have much smaller areas of honeycomb (1U, 3U, etc) that you move around the wall control board instead like you would with any other wall control part. 

My initial mindset has changed from "how can I add honeycomb to my wall control panels" to more of  "what is the best way to attach honeycomb designed elements to a wall control panel".  And I think that ends up being a slightly different problem with a different solution.  I *think* instead of wanting to make NxN honeycomb you want to make honeycomb in various Nx1/1xN combinations for horizontal and vertical mounting. 

I have some more wall control boards on order and once I have them, I can do some layouts with the board horizontal on a work surface and more accurately describe what is needed.

**Comment by boc-tothefuture** [comment]
I have been messing around making a few different center piece/HSW adapters and fine tuning the fit.   Would it be possible to make 1x0 Flat left/right pieces?

**Reply by aderusha** [reply]
It would!

https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/1x0%20Flat%20Left.stl

https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/1x0%20Flat%20Right.stl

**Reply by boc-tothefuture** [reply]
Awesome, I will print those and let you know.. I found some out in the wild (for something else) and the pin fit was a bit too tight.  

Do you want a PR for the adapters I am making?  I am doing them in openscad.

**Reply by aderusha** [reply]
Sure, or post them on Printables as a remix or whatever works for you. Hope the models work for ya!
